.github/workflows/check-catalyst.yaml

```diff
@@ -75,6 +75,8 @@ jobs:
           QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
           QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
           ENABLE_LIGHTNING_KOKKOS=ON \
+          CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" \
```

Comment:
I figured we should enable this in the CI as well, what do you think @maliasadi ?

Reply:
As we decided to build the wheels with OMP kernels, let's update the CI to be compatible with the wheels.